test/mocks/mock.data.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable quotes, quote-props, max-len */
 import constants from '../constants.js';
+import {versionedCredentials} from './credential.js';
```

Comment:
Looks like this change is missing from this PR.

Reply:
Nevermind. I realized this was a PR against the `update-vc-2.0` branch (and not `main`):
https://github.com/digitalbazaar/vc/blob/update-vc-2.0/test/mocks/credential.js#L6